data/mods/Magiclysm/items/alchemy_items.json

```diff
@@ -31,5 +31,74 @@
     "to_hit": -2,
     "qualities": [ [ "BUTCHER", -50 ] ],
     "//": "Potion recipes that require a full owlbear egg could also instead require larger amounts of stone shell. Since 1 egg gives 3 shells, you could perhaps have 1 egg be equivalent to 5 or more shells when making potions."
+  },
+  {
+    "id": "glow_dust",
+    "type": "GENERIC",
+    "symbol": "*",
+    "color": "yellow",
+    "name": "glow dust",
+    "description": "The powdered remains of a will-o-wisps's phsyical form.  It seems to still possess an otherworldly glow.",
+    "material": "powder",
+    "weight": 5,
+    "volume": 1
```

Comment:
a weight of 5 grams for a volume of 250mL is this what you intended?

Reply:
I based it on one of the other powders, like how coffee powder has a volume of 1 and weight of 14. I was thinking it'd be very ethereal and light.